.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,53 @@
+name: Node benchmarks
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 0 1 * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10
+    - name: benchmarks
+      run: |
+        npm install
```

Comment:
This is error prone and hard to manage. Put it in a separate file and let it be part of the linter.

Reply:
@StarpTech I have removed the inline node code to run the benchmarks, this required a fix to `benchmark-bench.js` to avoid the prompts.

Reply:
My idea is to move the entire code in a file and just run e.g `scripts/bench.js`